### ISSUE TITLE
Set FSW to be correct on model selection

### ISSUE THREAD

**Issue by Jedsters** [FrSkyRC/ETHOS-Feedback-Community#4556]
Can I question whether #2384  has really been addressed by #4161  which has been closed?
 I've just tried with 1.5.15 and if I have s.g. SF5 On and turn off the radio, when I turn it on again then SF5 is no longer active. 
I believe there should be an option for each model as to whether this behaviour should happen or not or better still a checklist option which sets the SF as desired rather than just notifying whether they are incorrect.
Thanks

**Comment by mawzthefinn** [comment]
Setting Persistent to On in Edit model will solve the first issue you note, it makes FSW's persistent through a reboot.

**Comment by Jedsters** [comment]
Thx, yes.

**Comment by Jedsters** [comment]
Also see #4560 which would be a pre-req to FS being automatically set as per the checklist settings.

**Comment by bsongis-frsky** [comment]
We could add some default position(s) settings which could be used:
1) on Flight Reset
2) on Model load, when "Persistent" is OFF

Then I wonder what would be the purpose of the Preflight Check warning about FS ...

**Comment by Jedsters** [comment]
Not quite sure I'm with you. Are you suggesting that default positions should be different to just setting FS to be as per the checklist at model load? If so, then as you say, checklist for FS wouldn't really serve a purpose. I was more thinking of using the checklist as the default positions which would be set on load if that was specified in the model definition.

On flight reset is an interesting thought and possibly valid. 

Setting the FS to be as per the checklist should IMHO be an alternative to persistent. I'm not sure I'd want to suggest that they should be the only 2 options, I suspect that some may like the current setup of just being alerted if any of the FS are not as per the checklist without it actually being corrected for them. 

Possibly 3 model definition load FS options

Checklist on model load
Persistent
Set to checklist settings

I think that the first one should result in the checklist FS options being checked. The second one I'm undecided and the last one not as it is irrelevant.

**Comment by mawzthefinn** [comment]
In my opinion, the Checklist should always learn from model settings, not vice versa. Any other option is inconsistent with the rest of the Checklist experience.

You set the plane up the way you want it, then learn those settings in the Checklist so it respects what you've set. It's not a setting screen for other functions

**Comment by Jedsters** [comment]
Hi Mawz, I think I must be misunderstanding you. 
Are you saying that for the FS the checklist should inherit its settings from how the FS are set by the user in the model? That doesn't make sense to me as the FS will change according to potentially what stage of a flight a user is in. I think there must be more going on than I'm considering. To my mind the checklist is this is how I want the FS when I load the model which only I can know. Whether the checklist inherits this from some other new setting in the model or the model sets it as per the current checklist I'm not too concerned with, although having another setting for the FS setup on model load seems a tad unnecessary IMHO, but as I say, that wouldn't bother me too much. Thanks

**Comment by mawzthefinn** [comment]
Yes, the Checklist should learn the settings from how the model is setup, the same way Switch positions are learned now.

You set the desired 'on start' positions then have the Checklist learn them. 

If you want to dictate which positions are active on model start, that should be a setting in Edit Model.

**Comment by Jedsters** [comment]
ok, that makes more sense and I'm sure will become fully clear in due course. Thanks!
